Package:     capi4hylafax
Subject:     use standard log directory
Author:      Joachim Wiedorn <ad_debian at joonet.de>
Forwarded:   no
Last-Update: 2012-06-13

Because of issues with hylafax (server) package the logfile
of c4h should go into the default /var/log/ directory.
---

diff -urN s06/config.faxCAPI s07/config.faxCAPI
--- s06/config.faxCAPI	2012-06-13 21:27:57.297794158 +0200
+++ s07/config.faxCAPI	2012-06-13 21:30:48.362179495 +0200
@@ -97,7 +97,7 @@
 #
 # default   "/tmp/capifax.log"
 #
-LogFile:                /var/spool/hylafax/log/capi4hylafax
+LogFile:                /var/log/hylafax/capifax.log
 
 
 # Der Befehl "LogTraceLevel" gibt die Menge der Daten an, die in ein LogFile
